src/config/config.ts

```diff
@@ -0,0 +1,131 @@
+import parse from 'parse-duration';
+import Web3 from 'web3';
+
+export interface Config {
+  chainConfigs: aloeChainConfig[];
+  multicallAddress: Address;
+  createAccountTopicID: string;
+  initialDeploy: number;
+  pollingInterval?: number;
+  processLiquidatableInterval?: number;
+  heartbeatInterval?: number;
+  heartbeatTimeout?: number;
+  clientKeepAliveTimeout?: number;
+  sanityCheckInterval?: number;
+  reconnectDelay?: number;
+  reconnectMaxAttemmpts?: number;
+  errorThreshold: number;
+  restartTimeout?: number;
+}
+
+interface aloeChainConfig {
+  chainName: string;
+  chainNumber: number;
+  factory: Address;
+  borrowerLens: Address;
+  borrower: Address;
+}
+
+type Address = string;
+type ConditionFunc<T> = (input: T) => boolean;
+
+const defaultTimeUnit = 'millisecond';
+const emptyString = '';
+
+const greaterThanZero = (input: number | undefined) => {
+  return input !== undefined && input > 0;
+};
+const greaterThanOrEqualToZero = (input: number | undefined) => {
+  return input !== undefined && input >= 0;
+};
+const isAddress = (input: string, chainNumber?: number) => {
+  return input !== emptyString && Web3.utils.isAddress(input, chainNumber);
+};
+const isNotEmpty = (input: string) => {
+  return input !== emptyString;
+};
+
+export function readConfig(json: any): Config | null {
```

Comment:
It would be better to define an interface here that describes the shape of the expected input to avoid using any.